services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -276,15 +276,16 @@ async def create_file_upload_links(
                 conn, parse_obj_as(SimcoreS3FileID, file_id)
             )
 
-        # ensure file is deleted first in case it already exists
-        await self.delete_file(
-            user_id=user_id,
-            file_id=file_id,
-            # NOTE: bypassing check since the project access rights don't play well
-            # with collaborators
-            # SEE https://github.com/ITISFoundation/osparc-simcore/issues/5159
-            enforce_access_rights=False,
-        )
+        if not is_directory:
+            # ensure file is deleted first in case it already exists
+            await self.delete_file(  # NOTE: MD check
```

Comment:
I guess you can remove the check
and maybe add a note with a link to the PR/issue we checked together? and that this is not needed for directories that are synced via an external tool.